### DISCUSSION TITLE
Zigbee2mqtt e dispositivi che si disconnettono.

### DISCUSSION THREAD

**Discussion by faustozago** [indomus/forum#2533] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno a tutti,
ho configurato una piccola rete zigbee (Z2M + Mqtt mosquitto , il tutto su HASSIO) con i seguenti dispositivi:
Coordinator : sonoff dongle plus V2-E
Sensori temperatura, pir ecc vari (quasi tutti SONOFF)
Sensori di presenza VISONIC MP-840 (compatibili con Z2M)
Due  sonoff  ZBMINI , che potrebbe funzionare da router .
Tutto funziona correttamente , anche se riavvio H.A.  con “RIAVVIO RAPIDO”  e “RIAVVIA HOME ASSISTANT”. 
Se invece faccio un “RIAVVIO DEL SISTEMA” oppure a casa salta la corrente e poi torna ,  alcuni dispositivi End Device (sia SONOFF che VISONIC ) , si disconnettono dal COORDINATOR ( sia che siano accoppiati alla SONOFF DONGLE V2 .E , sia siano accoppiati  ad uno dei SONOFF ZIGBEE MINI che fanno da ROUTER).
In altre parole , quando il router/coordinator si spegne ,  alcuni dispositivi end device non si connettono piu’ (sia SONOFF che altre marche).A volte 2 di questi dispositivi (sensore presenza VISONIC )  me li ritrovo connessi dopo 3-4 giorni, senza aver fatto niente, mentre dei sensori di temperatura SONOFF ,sono proprio spariti dalla mappa.
Altra prova che ho fatto è accoppiare i sensori  ai sonoff ZBMINI , lasciandoli sempre alimentati; ho spento completamente il raspberry (done è anche inserita la SONOFF DONGLE) . Al riavvio ha funzionato tutto correttamente , tutti i sensori collegati. Successivamente ho tolto la corrente dai due Sonoff ZBMINI , e si è presentato il problema, i sensori non si sono piu’ accoppiati-
A qualcuno è successo un problema del genere? 
Ho gia’ provato a cambiare canale (ora sono sul 25 , la wifi sull’1, ma non cambia niente).

Grazie

**Comment by faustozago** [comment]
RISOLTO:
due settimane fa ho sostituito il sonoff dongle-E con il "vecchio" sonoff dongle-P.
Tutto ha  funzionato correttamente da subito, ho aggiunto anche altri dispositivi, ho fatto tutti i test possibili (riavvio da mancanza corrente, riavvio normale completo, riavvio z2M , tutti per piu' volte , e ha sempre riaccoppiato tutti i dispositivi.
All'aquisto ho chiamato  il venditore del dongle, per assicurarmi che fosse la versione-P  (dall'inserzione non era chiaro) , mi ha detto che la dongle -E non la tiene piu' perche' ha un sacco di problemi e riceve un sacco di lamentele.
Probabilmente ha bisogno di un aggiornamento firmware serio....infatti sil sito Z2M non viene indicata come dispositivo compatibile  ma SPERIMENTALE, dove non viene garantita la stabilita'.

**Reply by KC-inDomus** [reply]
"sperimentale" solo perché il firmware non lo fa lui. ZBDongle-E è un roccia e funziona perfettamente con zigbee2mqtt e ZHA. non diffondiamo false informazioni per sentito dire.

**Reply by faustozago** [reply]
Scusate, non intendevo assolutamente dire che i sonoff non sono " buoni" , io stesso sono un grande fan di SONOFF , cerco altro solo se SONOFF non lo produce...
